modules/custom/az_migration/src/Plugin/migrate/process/MediaGenerate.php

```diff
@@ -0,0 +1,78 @@
+<?php
```

Comment:
For this process, I wonder if it would be better to migrate media first and then reference it later, rather than generating media on the fly?

https://www.drupal.org/project/media_migration

Reply:
I agree with this. I think Migration of media entities should be a separate migration that the person migration depends on, with resolution via `migration_lookup` plugin.

Reply:
Migrate all media entity is dumping all unwanted media into the migrated site. It is good to fetch only the wanted media that relevant/wanted for the content and map it. Thanks